frontend/src/scenes/experiments/ExperimentView/components.tsx

```diff
@@ -58,6 +62,78 @@ import { getIndexForVariant } from '../legacyExperimentCalculations'
+                    `Create a feedback survey comparing variants in the "${experiment.name}" experiment targeting users with feature flag "${featureFlagKey}"`,
+                    // Include specific variant suggestion only if variant exists
+                    ...(variants[0]?.key
+                        ? [
+                              `Create a survey for users who saw the "${variants[0].key}" variant of feature flag "${featureFlagKey}" in the "${experiment.name}" experiment`,
+                          ]
+                        : []),
+                    `Create an A/B test survey asking users to compare variants from feature flag "${featureFlagKey}" in the "${experiment.name}" experiment`,
+                    `Create a survey to understand which variant of feature flag "${featureFlagKey}" performed better in the "${experiment.name}" experiment`,
+                    `Create a survey targeting users exposed to any variant of feature flag "${featureFlagKey}" to gather feedback on the "${experiment.name}" test`,
+                ]
+              : [
+                    `Create a feedback survey for users who were exposed to feature flag "${featureFlagKey}" in the "${experiment.name}" experiment`,
+                    `Create an NPS survey for users who saw feature flag "${featureFlagKey}" during the "${experiment.name}" experiment`,
+                    `Create a satisfaction survey asking about the experience with feature flag "${featureFlagKey}" in the "${experiment.name}" experiment`,
+                    `Create a survey to understand user reactions to changes introduced by feature flag "${featureFlagKey}" in the "${experiment.name}" experiment`,
+                ],
+        context: {
+            user_id: user?.uuid,
+            experiment_name: experiment.name,
+            experiment_description: experiment.description,
+            feature_flag_key: experiment.feature_flag?.key,
+            feature_flag_id: experiment.feature_flag?.id,
+            feature_flag_name: experiment.feature_flag?.name,
+            target_feature_flag: experiment.feature_flag?.key,
+            survey_purpose: 'collect_feedback_for_experiment',
+            has_multiple_variants: hasMultipleVariants,
+            variants: variants.map((v) => ({
+                key: v.key,
+                name: v.name || '',
+                rollout_percentage: v.rollout_percentage,
+            })),
+            variant_count: variants?.length || 0,
+        },
+        callback: (toolOutput: { survey_id?: string; survey_name?: string; error?: string }) => {
+            if (toolOutput?.error || !toolOutput?.survey_id) {
+                posthog.captureException(toolOutput?.error, {
+                    source: 'survey-creation-failed',
+                    feature: 'surveys',
+                })
+                return
+            }
+            // Redirect to the new survey
+            router.actions.push(urls.survey(toolOutput.survey_id))
```

Comment:
let's make sure to capture the product intent action of survey created here, similar to how we do it for empty states / survey form / max ai surveys, this way we can know if those features are being used succesfully too